pkgs/stdenv/generic/default.nix

```diff
@@ -84,6 +84,9 @@ let
           ../../build-support/setup-hooks/set-source-date-epoch-to-latest.sh
           ../../build-support/setup-hooks/strip.sh
         ]
+        ++ lib.optionals hostPlatform.isX32 [
+          ../../build-support/setup-hooks/move-libx32.sh
+        ]
```

Comment:
Would it not be better to combine this with the existing move-lib64 hook, since they do basically the same thing?

Reply:
It would, but it would also rebuild the world for something rather inconsequential. I wasn't sure which one to use, so I used the solution that would make it convenient for me to make this work.

Reply:
I think we can do it at a later time unless there are critical changes which which cannot prevent a staging rebuild.

Reply:
I would be okay with making this PR into a draft for development for now, if it would make more sense to merge only the "final form" into nixpkgs